display/ssd1306.h

```diff
@@ -488,7 +493,7 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, Sa
     }
   }
 
-  void SetMessage(const char* text) {
+  void SB_Message(const char* text) {
```

Comment:
Just have SB_Message() call SetMessage instead.
Also: override.

Reply:
void SB_Message(const char* text) override {
    SetMessage(message_);
  }
  
  In this file doesn't work. Hint?

Reply:
Need to see the error message to give a hint.
(Did you put SetMessage() back?)

Reply:
I reverted all SB_Messages in the file back to SetMessage, yes.
Testing by sending `sd_card_not_found` in Serial Monitor.
No error message, just nothing displayed. Actually, I think I see garbage on the screen.
However, `setmessage hello`  does work. `setmessage hello\nworld` also, but line return ignored.

Reply:
Maybe I missed adding a DoMessage in saber_base?

Reply:
and just tested flipping all SetMessages in here to SB_Message, things work.